restli-tools/src/main/java/com/linkedin/restli/tools/clientgen/RestRequestBuilderGenerator.java

```diff
@@ -64,10 +64,19 @@
   public static void main(String[] args)
       throws IOException
   {
+    String[] sources = new String[0];
+
     if (args.length < 2)
     {
       _log.error("Usage: RestRequestBuilderGenerator targetDirectoryPath [sourceFile or sourceDirectory]+");
       System.exit(1);
+    } else if (args.length == 2 && args[1].startsWith("@")) {
```

Comment:
Similar comment as the one above, there should be a utility class within `:restli-tools` which handles the parsing side of things. Although this logic here isn't too complicated, this will proactively keep this sort of stuff from exploding if we need to extend or reuse it, plus it keeps this class clean.

Reply:
Added class `ArgumentFileProcessor`